tests/from.bats

```diff
@@ -345,3 +346,24 @@ load helpers
   run_buildah --log-level=error containers -f id=${cid}
   buildah rm ${cid}
 }
+
+@test "from with/without pull test" {
+  run buildah from --pull=false docker.io/nginx
+  [ $status -ne 0 ]
+  run buildah from --name nginxc docker.io/nginx
+  expect_output --substring "Getting"
+  run buildah from --name nginxc-1 docker.io/nginx
+  [[ "$output" != "Getting"* ]]
+  run buildah from --name nginxc-2 --pull-always docker.io/nginx
+  expect_output --substring "Getting"
+  buildah commit nginxc fakename-img
+  run buildah buildah from --pull-always fakename-img
+  [ $status -ne 0 ]
+  buildah rm nginxc nginxc-1 nginxc-2
+  buildah rmi fakename-img nginx
+}
+
+@test "from quiet test" {
+  run buildah from --quiet docker.io/busybox
+  [ "$output" != "Getting"* ]
```

Comment:
This is not a valid test, because (1) this-does-not-do-what-you-think-it-does, and (2) you don't know if busybox is already cached locally, in which case there would be no `Getting` message. A proper test would ensure that the desired image is not already present. This might be an instance in which `rmi` is necessary, much as I dislike that.

Reply:
Actually I should use [[ ]] here, and it should works.  Will update it. And yes, when I right a test case I default think the env is clean, so the image check is not request here. But as we want to keep the common use images during all tests, I will add the steps for exist images for it. Thanks for your comments.